### ISSUE TITLE
Import custom variables for single pages

### ISSUE THREAD

**Issue by sstaub** [bitfocus/companion#2493]
### Is this a feature relevant to companion itself, and not a module?

- [X] I believe this to be a feature for companion, not a module

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the feature

When I import a single page, the definitions for custom variables which are used at this page should also imported.

### Usecases

In the moment you have to add the custom variables used by a page manually after importing a page.

**Comment by geckigek** [comment]
Would be great to have that feature. Ran into the same problem.

+1

**Comment by semenovnick** [comment]
Yes - it's great. On that moment I can export only custom variables, but I can not import them without full reset... 
+1
